docs/user-guide/creating-application/workflow/plugins/copy-container-image.md

```diff
@@ -0,0 +1,67 @@
+## Copy Container Image
+
+This plugin copies a container image from the source to destination. The pushing of image can be between repositories of the same container registry or between repositories of different container registry.
```

Comment:
Suggestion: This plugin copies a container image to a desired container repository.